WPFHospitalEditor/AlternativeRelocationAppointments.xaml

```diff
@@ -0,0 +1,20 @@
+﻿<Window x:Class="WPFHospitalEditor.AlternativeRelocationAppointments"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:WPFHospitalEditor"
+        mc:Ignorable="d"
+        Title="AlternativeRelocationAppointments" Height="150" Width="340" WindowStartupLocation="CenterScreen">
+    <StackPanel HorizontalAlignment="Stretch">
+        <StackPanel Orientation="Vertical" HorizontalAlignment="Stretch" Height="50" Margin="10,10,10,0" >
+            <TextBlock x:Name="givenRoom" TextAlignment="Center" HorizontalAlignment="Center">Room with id: </TextBlock>
+            <TextBlock HorizontalAlignment="Center" TextAlignment="Center" TextWrapping="WrapWithOverflow">is unavailable in given time interval. Click 'Show on map' to see that room.</TextBlock>
```

Comment:
You can use one text block here, and set it's value with given id in constructor.

Reply:
Now only one textBlock exists, and it's value is setting in constructor.